tekton/release-cheat-sheet.md

```diff
@@ -33,32 +27,59 @@ the pipelines repo, a terminal window and a text editor.
     git show $TEKTON_RELEASE_GIT_SHA
     ```
 
-1. Create a workspace template file:
+1. Execute the release PipelineRun (takes ~45 mins).
 
-   ```bash
-   cat <<EOF > workspace-template.yaml
-   spec:
-     accessModes:
-     - ReadWriteOnce
-     resources:
-       requests:
-         storage: 1Gi
-   EOF
-   ```
-
-1. Execute the release pipeline (takes ~45 mins).
+    **If you are back-porting include this in the params list:**
+    ```yaml
+    - name: releaseAsLatest
+      value: false
+    ````
 
-    **If you are back-porting include this flag: `--param=releaseAsLatest="false"`**
+    ```bash
+    cat <<EOF > release-pipelinerun.yaml
+    apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      generateName: release-pipelinerun-
+    spec:
+      serviceAccountName: release-right-meow
+      params:
+        - name: gitRevision
+          value: ${TEKTON_RELEASE_GIT_SHA}
+        - name: serviceAccountPath
+          value: release.json
+        - name: versionTag
+          value: ${TEKTON_VERSION}
+        - name: releaseBucket
+          value: gs://tekton-releases/pipeline
+      workspaces:
+        - name: release-secret
+          secret:
+            secretName: release-secret
+        - name: workarea
+          volumeClaimTemplate:
+            spec:
+              accessModes:
+              - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 1Gi
+      pipelineRef:
+        resolver: git
+        params:
+          - name: org
+            value: tektoncd
+          - name: repo
+            value: pipeline
+          - name: revision
+            value: main
```

Comment:
Shouldn't this be the SHA of the release and not necessarily main?

Reply:
That makes more sense. Change!
I thought it's release pipeline yaml, which is not necessarily to be the same as the release SHA.